### PR TITLE
Button for shortcuts

### LINKED ISSUE #1233: Add a button for help `?` that opens the keyboard shortcuts
After we merge #1224 

Probably next to the language select

### PR DESCRIPTION
**Pull request by mbj36**
Fixes #1233

### CODE REVIEW

**Comment by lipis** [comment]
It looks quite ugly.. we need to think this through.

**Comment by dwelle** [comment]
Yes, icon-only + ensure i thas the same height as the language picker.

**Comment by lipis** [comment]
Another idea would be to replace the GitHub corner with a questionmark/help corner. Just an idea /cc @dwelle @vjeux

**Comment by mbj36** [comment]
@lipis I converted to icon only. 
Screenshot 2020-04-06 at 5 27 26 PM

**Comment by vjeux** [comment]
Let’s keep the github corner for now. It’s been very useful to get people to contribute

File: src/styles.scss [hunk 1]
```diff
@@ -410,6 +410,12 @@ button,
   padding: 10px 20px;
 }
 
+.help-icon {
+  position: fixed;
+  bottom: 14px;
+  right: 15px;
```
Comment (lipis) [inline src/styles.scss:416]:
```suggestion
  right: 10px;
  fill: #868e96;
```

**Comment by vjeux** [comment]
What about using a keyboard icon since it shows keybindings?
CA201605-B340-4FFC-9E1B-355662539E64

**Comment by dwelle** [comment]
It doesn't show *only* keybindings, albeit it's its main purpose, yea.

**Comment by vjeux** [comment]
I like that last iteration

**Comment by j-f1** [comment]
This looks misaligned to me but maybe it’s just an optical illusion:

Screen Shot 2020-04-06 at 14 26 14

**Comment by j-f1** [comment]
Also don’t like this space:

Screenshot_2020-04-06 14 27 14

I don’t think the rows need a `title` attribute. We should show for the shape options that you can press <kbd>S</kbd> **or** <kbd>1</kbd> because people might assume you have to press <kbd>S</kbd> then <kbd>1</kbd>.

**Comment by lipis** [comment]
@j-f1 Yeap.. valid points! Could you file an issue, as it should be done probably in a separate PR, as this is just for the button.

**Comment by mbj36** [comment]
@lipis Can this be merged ?

**Comment by dwelle** [comment]
I'll make a PR for the label changes.

**Comment by lipis** [comment]
and I'll fix the `or` for shortcuts..

**Comment by dwelle** [comment]
> and I'll fix the `or` for shortcuts..

That's what I meant.. but go ahead :)

**Comment by lipis** [comment]
I was already doing some other changes in the shortcuts as well.. so let me finish it.

**Pull request merged**